sdk/react/ooyalaSkinBridgeListener.js

```diff
@@ -19,7 +19,7 @@ export default class OoyalaSkinBridgeListener {
     Log.log('SkinBridgeListener Mounted');
     this.listeners = [];
     const listenerDefinitions = [
-    /* eslint-disable */
+    /* eslint-disable no-multi-spaces */
```

Comment:
Why disable this rule? Maybe better format it according to the code style?

Reply:
Well, those space/tabs were intentionally added for better visibility.
We can omit them but I'd rather left them in some similar way. Anyway, if you can propose a better formatting I'll update it

Reply:
@ooyala-amohnatkin this rule was removed